cmd/protecodeExecuteScan.go

```diff
@@ -160,6 +161,12 @@ func getURLAndFileNameFromDockerImage(scanImage string, registryURL string, file
 	completeURL := scanImage
 
 	if len(registryURL) > 0 && len(filePath) <= 0 {
+
+		url, _ := url.Parse(registryURL)
+		if len(url.Scheme) > 0 {
+			registryURL = strings.Replace(registryURL, fmt.Sprintf("%v://", url.Scheme), "", 1)
```

Comment:
URL has the notion of `protocol://...`

We should not have a URL without protocol.
stripping the protocol off gives us the registry ...